perses/analysis/free_energies.py

```diff
@@ -0,0 +1,242 @@
+import numpy as np
+import seaborn as sns
+from pymbar import BAR
+import matplotlib.pyplot as plt
+import seaborn
+import pickle
+from simtk.openmm import unit
+import bz2
+import json
+import tqdm
+from openmmtools.constants import kB
+import random
+import joblib
+from typing import Optional
+
+
+def _strip_outliers(w, n_devs=100):
+    w = [x for x in w if np.abs(x) < 10 ** 4]
```

Comment:
Is `w` a `np.ndarray`? In that case you can use masking like this:

```python
w = w[w.abs() < 1E4]
```

Reply:
Same in similar comprehensions below.

Reply:
Thanks, fixed in 78fd847.